### DISCUSSION TITLE
Pause Script?

### DISCUSSION THREAD

**Discussion by cyberomak** [pfelk/pfelk#542] [Ideas]
Anyway to pause the install script after the elasticsearch install to modify the jvm.options settings? Otherwise, the script stops because ES errors out, then you modify jvm.options, then have to start again.

**Comment by a3ilson** [comment]
@cyberomak thank you for providing this feedback...I'll test the script, to attempt to replicate the issue and resolve. 

- What OS are you running the script on?
- How much memory (ram) does the target system have allocated?

**Reply by cyberomak** [reply]
Running it on Debian 12. Though it is the Proxmox LXC image, so not sure if that matters too much. 

I have 14GB of RAM assigned to the VM now. But it doesn't matter how much I assign, it can never read it and then errors out on starting up Elasticsearch until I go in and manually modify it. Maybe another idea would be to add an option into the script where the user can put in what they want and it modifies the jvm.options file then attempts to start ES?

**Comment by a3ilson** [comment]
The script does not modify or touch the jvm.options file...using the defaults from the package.

I'll test the script on an LXC image in the near future...I'm not sure if it is related to the jvm.options file or something else.
